docker-compose-dev.yml

```diff
@@ -66,8 +64,6 @@ services:
       command: --copy-service
     phpldapadmin:
       image: osixia/phpldapadmin
-      links:
-        - openldap:ldap-host
       ports:
         - '${FORWARD_PHPLDAPADMIN_PORT:-6680}:80'
       environment:
```

Comment:
Please also adjust the environment:
`PHPLDAPADMIN_LDAP_HOSTS: "#PYTHON2BASH:[{'Local LDAP': [{'server': [{'host': 'openldap'}]},{'login': [{'bind_id': 'cn=admin,dc=university,dc=org'},{'bind_pass': 'admin'},{'auth_type': 'config'}]}]}]"`

Reply:
If you use the configuration I commented on, the users are immediately logged into the local LDAP administration area.
I tested this config on my dev pc